### PR TITLE
update chinese comment

### PR DESCRIPTION
**Pull request by jianglong0156**
update chinese comment

### CODE REVIEW

**Comment by jianglong0156** [comment]
@folgerhu Please review it

File: cocos/platform/CCImage.h [hunk 1]
```diff
@@ -94,30 +97,39 @@ class CC_DLL Image : public Ref
...
+    @return @~english true if loaded correctly.
+     * @~chinese 如果正确加载则返回true
     * @js NA
     * @lua NA
     */
     bool initWithImageData(const unsigned char * data, ssize_t dataLen);
 
-    // @warning kFmtRawData only support RGBA8888
+    /** @~english Load image from raw data @~chinese 从原始数据中加载图片
+    // @warning kFmtRawData @~english only support RGBA8888 @~english 只支持RGBA8888
```
Comment (folgerhu) [inline cocos/platform/CCImage.h]: @~chinese

File: cocos/platform/CCImage.h [hunk 2]
```diff
@@ -202,11 +224,15 @@ class CC_DLL Image : public Ref
     Image & operator=(const Image&);
     
     /*
-     @brief The same result as with initWithImageFile, but thread safe. It is caused by
+     @brief @~english The same result as with initWithImageFile, but thread safe. It is caused by
      loadImage() in TextureCache.cpp.
-     @param fullpath  full path of the file.
-     @param imageType the type of image, currently only supporting two types.
-     @return  true if loaded correctly.
+     * @~chinese 它和使用initWithImageFile的结果是一样的，只不过线程相对安全。它在源文件TextureCache.cpp中被 loadImage()调用
```
Comment (folgerhu) [inline cocos/platform/CCImage.h]: 感觉怪怪的，是否可译成：它被TextureCache.cpp中的 loadImage()所调用

**Comment by jianglong0156** [comment]
@folgerhu I improve the chinse comment Please review it

**Comment by jianglong0156** [comment]
@andyque  It's ok

**Comment by folgerhu** [comment]
@andyque review over, it's ok

**Pull request merged**